client/my-sites/plugins/marketplace/marketplace-stand-alone-thank-you/index.tsx

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
What's the value of this function, considering that it is a sole wrapper and provides nothing on top of that?

Reply:
I was actually planning to move that component in here since marketplace seems to require its own thank-you structure without going through checkout.